cli/package.json

```diff
@@ -13,7 +13,7 @@
     "size": "t=\"cypress-v0.0.0.tgz\"; yarn pack --filename \"${t}\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
     "test": "yarn test-unit",
     "test-debug": "node --inspect-brk $(yarn bin mocha)",
-    "test-dependencies": "dependency-check . --no-dev",
+    "test-dependencies": "dependency-check . --missing --no-dev --verbose",
```

Comment:
`dependency-check` changed their default args in 4.0, so this should be equivalent to what it used to be.